app/services/force/custom_api/application_service.rb

```diff
@@ -47,9 +47,16 @@ def application_snapshot(id)
         application
       end
 
-      def submit(custom_api_attrs)
+      def submit(custom_api_attrs, method)
         application_params = application_defaults.merge(custom_api_attrs)
-        result = api_post('/LeasingAgentPortal/shortForm', application_params)
+        application_params = application_params.except('primaryApplicant') if application_params['primaryApplicant'].blank?
```

Comment:
Why are this and the exceptions on line 56 necessary?

Reply:
`primaryApplicant` gets pulled in short_form_controller.rb#10 but it's empty, so to avoid messing with SF I'm removing it here but only if we are not actually updating it. Regarding line 56, we dont't want to edit them and I recall having SF errors when they were present. On the other hand we want those fields for POST request, so we can't filter them on controller params.